File: stephanie/Problema0305/Problema0305.js
const prompt = require("prompt-sync")({ sigint: true })

let vetor = []

for(let i = 0; i < 10; i++){
    vetor[i] = prompt("Digite um número: ")
}
let j = 0
while(vetor[j]>0 && j<10){
    j=j+1
}
if(j=9){
    console.log("Não existem valores negativos")
}else{
    console.log("O número negativo está na posição: "+j)
}

File: stephanie/Problema0305/Problema0305.java
package stephanie.Problema0305;

public class Problema0305 {

public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int[] vetor = new int[];
        for(int i;i < 10;i++){
            System.out.println("Digite um número: ");
            vetor[i] = input.nextLine();
        }
        int j = 0;
        while(vetor[j]>0 && j<10){
            j=j+1
        }
        if(j=9){
            System.out.println("Não existem valores negativos");
        }else{
            System.out.println("O número negativo está na posição: "+j);
        }
    }


}